documentation/modules/auxiliary/scanner/ip/ipidseq.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+
+This ```auxiliary/scanner/ip/ipidseq``` module will probe hosts' IPID sequences and classify them using the same method Nmap uses when it's performing its IPID Idle Scan (-sI) and OS Detection (-O).
```

Comment:
this section covers a lot of the differences between nmap and this module. It may be better to put that in its own L3 heading, or to do a table to break down the differences.